src/main/resources/processes/PSU/PSU_IEDET_COMPLAINT.bpmn

```diff
@@ -46,6 +46,11 @@
       <bpmn:outgoing>Flow_10v7ld8</bpmn:outgoing>
     </bpmn:exclusiveGateway>
     <bpmn:sequenceFlow id="Flow_0mvcbzc" sourceRef="Gateway_0n7b72e" targetRef="EndEvent_Complaint" />
+    <bpmn:serviceTask id="Service_AddCaseTag" name="Add a case tag" camunda:expression="${taggingService.createTagForCase(execution.getVariable(&#34;CaseUUID&#34;), &#34;REJECTED BY PSU&#34;)}">
```

Comment:
Could we change this to `REJECTED_BY_PSU` and add the mapping change on the FE to this ticket. It would also allow for us to put custom colours in as well and hit that AC.